desktop-themes/BlueMenta-dark/gtk-3.0/settings.ini

```diff
@@ -0,0 +1,4 @@
+[Settings]
```

Comment:
This file is generated - only the `settings.ini.in` file should be tracked in git.